metrics/web/test/features/dashboard/presentation/widgets/project_metric_circle_percentage_test.dart

```diff
@@ -0,0 +1,177 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/features/common/presentation/metrics_theme/model/circle_percentage_theme_data.dart';
+import 'package:metrics/features/common/presentation/metrics_theme/model/metric_widget_theme_data.dart';
+import 'package:metrics/features/common/presentation/metrics_theme/model/metrics_theme_data.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/circle_percentage.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/project_metric_circle_percentage.dart';
+
+import '../../../../test_utils/testbed_page.dart';
+
+void main() {
+  group("ProjectMetricCirclePercentage", () {
```

Comment:
What if we implement widget to accept ThemeStrategy (see https://en.wikipedia.org/wiki/Strategy_pattern) - thus we will be able to test widget separately from ThemeStrategy (it would be much simpler to test then)?